client/src/components/App/stories/index.story.js

```diff
@@ -9,67 +9,90 @@ const renderLink = (url, text) => (
   </a>
 );
 
-storiesOf('Welcome Story', module)
-  .add('MERN Chatbot', () => (
-    <div style={{ padding: paddingSize }}>
-      <h4>MERN Chatbot Storybook</h4>
+storiesOf('Welcome Story', module).add('MERN Chatbot', () => (
+  <div style={{ padding: paddingSize }}>
+    <h4>MERN Chatbot Storybook</h4>
 
-      <p>
-        This is the UI documentation of components for my presonal project
-        {' '}
-        {renderLink('https://github.com/KovDimaY/MERN-ChatBot/issues', 'MERN Chatbot Storybook')}
-      </p>
+    <p>
+      This is the UI documentation of components for my presonal project{' '}
+      {renderLink(
+        'https://github.com/KovDimaY/MERN-ChatBot/issues',
+        'MERN Chatbot Storybook'
+      )}
+    </p>
 
-      <p style={{ marginTop: marginSize }}>
-        I have connected many {renderLink('https://storybook.js.org/addons/', 'addons')} to
-        the project, so you can enjoy all the variaty of the features like:
-      </p>
+    <p style={{ marginTop: marginSize }}>
+      I have connected many{' '}
+      {renderLink('https://storybook.js.org/addons/', 'addons')} to the project,
+      so you can enjoy all the variaty of the features like:
+    </p>
 
-      <ul style={{ paddingLeft: paddingSize }}>
-        <li style={{ listStyleType: 'disc' }}>
-          Watching the info about usage and props of the component
-          ({renderLink('https://github.com/storybooks/storybook/tree/master/addons/info', 'addon-info')})
-        </li>
-        <li style={{ listStyleType: 'disc' }}>
-          Watching logs of the actions and events of the component
-          ({renderLink('https://github.com/storybooks/storybook/tree/master/addons/actions', 'addon-actions')})
-        </li>
-        <li style={{ listStyleType: 'disc' }}>
-          Watching the source code of the story of the component
-          ({renderLink('https://github.com/storybooks/storybook/tree/master/addons/storysource', 'addon-storysource')})
-        </li>
-        <li style={{ listStyleType: 'disc' }}>
-          Changing the viewport to see how the component looks like on different devices
-          ({renderLink('https://github.com/storybooks/storybook/tree/master/addons/viewport', 'addon-viewport')})
-        </li>
-        <li style={{ listStyleType: 'disc' }}>
-          Changing values of the props to see how it changes the component
-          ({renderLink('https://github.com/storybooks/storybook/tree/master/addons/knobs', 'addon-knobs')})
-        </li>
-      </ul>
+    <ul style={{ paddingLeft: paddingSize }}>
+      <li style={{ listStyleType: 'disc' }}>
+        Watching the info about usage and props of the component (
+        {renderLink(
```

Comment:
This change looks less pretty than it was before 😕

Probably I need to change the allowed length of the line in the prettier config 🤔

Reply:
FIXED with the length 100